lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -166,9 +166,18 @@ public int getMegaBytes() {
     private Builder() {
     }
 
+    // setting builder to the values passed in the request
     private Builder(final EvaluatorRequest request) {
       setNumber(request.getNumber());
       fromDescriptor(request.getDescriptor());
+      setMemory(request.getMegaBytes());
```

Comment:
These changes are a bugfix that isn't related to the change at hand, right? Maybe we should make it a separate JIRA?